conda_smithy/configure_feedstock.py

```diff
@@ -915,13 +900,13 @@ def _load_forge_config(forge_dir, exclusive_config_file):
                   'linux': 'circle', 
                   'osx': 'travis', 
                   'win': 'appveyor',
-                  'linux_aarch64': 'azure',
-                  'linux_ppc64le': 'azure',
+                  # Following platforms are disabled by default
+                  'linux_aarch64': None,
```

Comment:
technically the enabled switch is on line 910

Reply:
Those enabled switches are not supposed to be overridden by the users. They are used for the Readme etc. Overriding using
```
provider:
  ppc64le: azure
```
should work